pages/admin/dashboard/tools.tsx

```diff
@@ -27,11 +27,22 @@ const ToolsPage: React.FC = () => {
     // TODO: Need to update this to calculate relative date
     const date = new Date();
 
-    const filterTools = () => {
-        const filteredTools = toolsList.filter((t) => {
-            return t["status"] === selectedTab;
-        });
-        setToolsArray(filteredTools);
+    const filterTools = async () => {
+        try {
+            const response = await axios({
+                method: "GET",
+                url: "/api/tool/getAll",
```

Comment:
`/api/tool` calls `getAll`, so `url: "/api/tool"` would work

Reply:
Updated url to `/api/tool`